src/atoms/EditableTitle/EditableTitle.story.tsx

```diff
@@ -0,0 +1,11 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+
+import AddressLabelProps from './EditableTitle';
+
+storiesOf('Atoms', module).add('EditableTitle', () => (
+  <>
+    <AddressLabel editable={true} onSubmit={console.log} />
+    <AddressLabel editable={false} />
+  </>
+));
```

Comment:
I don't think we need this story for external developers, as the component is only used internally to implement EditableTable.